spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -25,27 +28,211 @@
       'SC0340, Accession 2005-101',
       'SC0340, Accession 2005-101, Box 18'
     ]
-    parsable_exemplars_based_on_actual_data.each do |example|
-      it "parses series number from '#{example}'" do
-        ng_mods = Nokogiri::XML("#{mods_start}#{example}#{mods_end}")
+    exemplars_based_on_feigenbaum.each do |example|
```

Comment:
there are definitely things to like about what you are suggesting.  I am happy to work with you tomorrow on improving this, tho I wonder if we want to make a separate PR for the refactoring so this can get "in" as a working thing first, and refactored after, as an additional task.

Also, I've started working on folder name (sul-dlss/sul-exhibits-template#52) which is a pre-req for  the display location (#18).  I think those two PRs will clash otherwise.